test/test_spin_gates.py

```diff
@@ -179,6 +181,118 @@ def test_lz2_gate(self):
                 )
             )
 
+    def test_rydberg_block_gate(self):
+        """check matrix form of the rydberg blockade gate on two qubits"""
+        chi = 2 * np.pi
+        circ = QuantumCircuit(2)
+        circ.append(RydbergBlockade(2, phi=chi), qargs=[0, 1])
+        # add gate to circuit via the @add_gate-decorated method
+        circ_decorated = QuantumCircuit(2)
+        circ_decorated.rydberg_block(chi, [0, 1])
+        for circuit in [circ, circ_decorated]:
+            unitary = self.backend.run(circuit).result().get_unitary()
```

Comment:
I did not know we could do this. If we are just testing the `RydbergBlockade` instruction could we do:
```python
from qiskit.quantum_info import Operator

np.allclose(Operator(RydbergBlockade(2, phi=chi)).data, expected)
```
Does that work?

Reply:
That being said, I also like how the decorated circuit is being tested too. Perhaps we can have an additional test using the Quantum Info. operator too? Maybe with different parameters?

Reply:
I think hat I was able to implement your suggestion.